python/semantic_kernel/memory/chroma/chroma_memory_store.py

```diff
@@ -0,0 +1,345 @@
+from logging import Logger
+from typing import TYPE_CHECKING, List, Optional, Tuple
+
+from numpy import array, ndarray
+from semantic_kernel.memory.chroma.utils import (
+    camel_to_snake, chroma_compute_similarity_scores, query_results_to_records)
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+from semantic_kernel.utils.null_logger import NullLogger
+
+if TYPE_CHECKING:
+    import chromadb
+    import chromadb.config
+    from chromadb.api.models.Collection import Collection
+
+
+class QdrantMemoryStore(MemoryStoreBase):
+    _client: "chromadb.Client"
+    _logger: Logger
+
+    def __init__(
+        self,
+        persist_directory: Optional[str] = None,
+        client_settings: Optional["chromadb.config.Settings"] = None,
+        similarity_fetch_limit: int = 20,
+        logger: Optional[Logger] = None,
+    ) -> None:
+        """
+        ChromaDataStore provides an interface to store and retrieve data using ChromaDB.
```

Comment:
All references to ChromaDataStore should be deleted. I think in most cases it can be replaced with either MemoryStoreBase or ChromaMemoryStore (context dependent)